pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/TupleSketchIntegrationTest.java

```diff
@@ -96,7 +95,8 @@ public void testQueries(boolean useMultiStageQueryEngine)
             DEFAULT_TABLE_NAME);
     JsonNode jsonNode = postQuery(query);
     assertTrue(jsonNode.get("resultTable").get("rows").get(0).get(0).asLong() > 0);
-    assertEquals(jsonNode.get("resultTable").get("rows").get(0).get(1).asText().length(), 1756);
+    assertTrue((jsonNode.get("resultTable").get("rows").get(0).get(1).asText().length() == 1740) || (
```

Comment:
This test is too brutal, and not really verify anything..
For the raw tuple sketch, we should just deserialize it and verify if it returns the same distinct count as the first value